### PR TITLE
Feature/add toast dialog

### PR DESCRIPTION
**Pull request by piloshachar14**

### CODE REVIEW

File: apps/client/src/components/add-toast/add-toasts.tsx [hunk 1]
```diff
@@ -36,17 +59,157 @@ export const AddToast: React.FC<Props> = ({
...
+    const {
+      target: { value },
+    } = event;
+    setSolidsPick(typeof value === 'string' ? value.split(',') : value);
+  };
+
+  const solids = [
+    'פלאפל',
+    'פיצות',
+    '',
```
Comment (TomerSteinberg) [inline apps/client/src/components/add-toast/add-toasts.tsx]: Was it intended for there to be an empty string here?

File: apps/client/src/components/add-toast/add-toasts.tsx [hunk 2]
```diff
@@ -36,17 +59,157 @@ export const AddToast: React.FC<Props> = ({
...
+    'גלידה',
+    'קרקרים ומטבלים',
+    'פירות',
+    'עוגיות',
+    'עוגות',
+  ];
+
+  const fluids = ['אייס קפה', 'שוקו', '', 'מיץ', 'סודה', 'מים בטעמים', 'קולה'];
```
Comment (TomerSteinberg) [inline apps/client/src/components/add-toast/add-toasts.tsx]: Was it intended for there to be an empty string here?

File: apps/client/src/components/add-toast/add-toasts.tsx [hunk 3]
```diff
@@ -36,17 +59,157 @@ export const AddToast: React.FC<Props> = ({
...
+                <Select
+                  labelId="demo-multiple-name-label"
+                  id="demo-multiple-name"
+                  multiple
+                  value={solidsPick}
+                  onChange={handleChange}
+                  input={<OutlinedInput label="Name" />}
+                >
+                  {solids.map((solid) => (
+                    <MenuItem key={solid} value={solid}>
```
Comment (TomerSteinberg) [inline apps/client/src/components/add-toast/add-toasts.tsx]: Are you sure the key needs to be `solid`.
What if you have the same solid twice? 
I think theres a better way..

File: apps/client/src/components/category/category.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { IoMdAddCircleOutline } from 'react-icons/io';
 import styles from './category.module.css';
 import React, { PropsWithChildren, useState } from 'react';
 import { Tooltip } from 'react-tooltip';
-import { AddToast } from '../add-toast/add-toasts';
+import { AddToast } from '..';
```
Comment (TomerSteinberg) [inline apps/client/src/components/category/category.tsx:5]: I don't think this is a valid import route

File: apps/client/src/components/add-toast/add-toasts.tsx (hunk 2 above)
Comment (piloshachar14) [reply apps/client/src/components/add-toast/add-toasts.tsx]: it meant to be a divider which later will be disabled

File: apps/client/src/components/add-toast/add-toasts.tsx (hunk 1 above)
Comment (piloshachar14) [reply apps/client/src/components/add-toast/add-toasts.tsx]: it meant to be a divider which later will be disabled

File: apps/client/src/components/add-toast/add-toasts.tsx (hunk 3 above)
Comment (piloshachar14) [reply apps/client/src/components/add-toast/add-toasts.tsx]: yes,index. noted

File: apps/client/src/components/category/category.tsx (hunk 4 above)
Comment (piloshachar14) [reply apps/client/src/components/category/category.tsx:5]: why?

Comment (TomerSteinberg) [reply apps/client/src/components/category/category.tsx:5]: After asking postol apparently this is valid although weird.
My bad though you can resolve

**Pull request merged**